recipes/librdkafka/all/patches/0002-Change-library-targets-and-result-variables-1-5-0.patch

```diff
@@ -0,0 +1,48 @@
+--- src/CMakeLists.txt	(revision 39796d359898c07ea422849e6d7cd34cd13ec466)
++++ src/CMakeLists.txt	(date 1596441634092)
+@@ -209,9 +209,9 @@
+ endif()
+ 
+ if(WITH_ZSTD)
+-  target_link_libraries(rdkafka PUBLIC ${ZSTD_LIBRARY})
+-  target_include_directories(rdkafka PUBLIC ${ZSTD_INCLUDE_DIR})
+-  message(STATUS "Found ZSTD: ${ZSTD_LIBRARY}")
++  target_link_libraries(rdkafka PUBLIC ${zstd_LIBRARY})
++  target_include_directories(rdkafka PUBLIC ${zstd_INCLUDE_DIR})
++  message(STATUS "Found ZSTD: ${zstd_LIBRARY}")
+ endif()
+ 
+ if(WITH_SSL)
+@@ -224,8 +224,8 @@
+     add_dependencies(rdkafka bundled-ssl)
+   else()
+     find_package(OpenSSL REQUIRED)
+-    target_include_directories(rdkafka PUBLIC ${OPENSSL_INCLUDE_DIR})
+-    target_link_libraries(rdkafka PUBLIC OpenSSL::SSL OpenSSL::Crypto)
++    target_include_directories(rdkafka PUBLIC ${OpenSSL_INCLUDE_DIR})
```

Comment:
```suggestion
```

If we are already using the target, the include-dir shouldn't be neccesary.

Reply:
Actually the generated definition from components follow the filename, which is OpenSSL, however, the required definition is upper-case. It should be the next improvement for components

Reply:
Maybe that's not something for `cpp_info`, but build-modules specific to the library: https://github.com/conan-io/conan/issues/7261

At some point in time, we need to stop adding features that are too much CMake-centric 😄

Reply:
I hope this should be the last major feature for cmake generators ... :sweat_smile:

Reply:
From my perspective, I'll try not to add anything else specific to CMake's related generators, so I'll try always to offer a workaround to any PR 👮‍♂️  ... until I get a proof it is something needed and common in many CMake libraries. 

Here, we could add a build_module to `openssl` and populate this all-uppercase variables with the value of the property from the targets, right? IMHO, this doesn't require another attribute in `cpp_info`.

I can be wrong, that's the reason why I confess my positioning